configure.ac

```diff
@@ -8,10 +8,10 @@
 #
 AC_PREREQ([2.63])
 
-define([ucx_ver_major], 1)  # Major version. Usually does not change.
-define([ucx_ver_minor], 14) # Minor version. Increased for each release.
-define([ucx_ver_patch], 0)  # Patch version. Increased for a bugfix release.
-define([ucx_ver_extra], "") # Extra version string. Empty for a general release.
+define([ucx_ver_major], 1)      # Major version. Usually does not change.
+define([ucx_ver_minor], 14)     # Minor version. Increased for each release.
+define([ucx_ver_patch], 0)      # Patch version. Increased for a bugfix release.
+define([ucx_ver_extra], "pre1") # Extra version string. Empty for a general release.
```

Comment:
maybe ".pre1" ?

Reply:
IMO subversion should not include separator, the same as `major`, `minor` and `patch`

Reply:
i think yes because "extraversion" is appended to regular version, see for example https://github.com/torvalds/linux/blob/master/Makefile#L5

Reply:
hm.. ok then

Reply:
returned back due to https://dev.azure.com/ucfconsort/7541fc4e-4907-45d4-ac29-8da7f8540a87/_apis/build/builds/42134/logs/583